markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 36 Line 6_
```markdown
```python
print "Hello World"
```

```javascript
console.log("Hello World")
```
Console log review comment 5

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._

Reply:
response to comment 5

Reply:
5